src/modules/sharing/sharing.module.ts

```diff
@@ -44,6 +46,6 @@ import { AppSumoModule } from '../app-sumo/app-sumo.module';
     SequelizeUserReferralsRepository,
     PaymentsService,
   ],
-  exports: [SharingService, SequelizeSharingRepository],
+  exports: [SharingService, SequelizeSharingRepository, SequelizeModule],
```

Comment:
Exporting SequelizeModule is recommended by Nestjs in order to prevent failing due to model dependencies if you use repositories in other modules. 

https://docs.nestjs.com/techniques/database#models

If you want to use the repository outside of the module which imports SequelizeModule.forFeature, you'll need to re-export the providers generated by it. You can do this by exporting the whole module, like this: